bpf/CMakeLists.txt

```diff
@@ -5,61 +5,59 @@ set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin)
 
-# Each test consists of a C file, an output file name, and an optional option, seperated by commas.
+# Each test consists of a C file, an output file name, and an optional option-list, seperated by commas.
 set(test_cases
-    "baseline"
+    "baseline,baseline,-DBPF"
     "generic_map,hash,-DTYPE=BPF_MAP_TYPE_HASH"
     "generic_map,percpu_hash,-DTYPE=BPF_MAP_TYPE_PERCPU_HASH"
     "generic_map,lru_hash,-DTYPE=BPF_MAP_TYPE_LRU_HASH"
     "generic_map,lru_per_cpu_hash,-DTYPE=BPF_MAP_TYPE_LRU_PERCPU_HASH"
     "generic_map,array,-DTYPE=BPF_MAP_TYPE_ARRAY"
     "generic_map,percpu_array,-DTYPE=BPF_MAP_TYPE_PERCPU_ARRAY"
-    "helpers"
+    "helpers,helpers,-DBPF"
```

Comment:
What's with the extra -DBPF here?

Reply:
In the previous implementation, `-DBPF` was added to all entries with one element, thus, porting the same behavior on the current one.